src/examples/adb_marionette_proxy.js

```diff
@@ -0,0 +1,34 @@
+import Testdroid from 'testdroid-client';
+let session;
+async () => {
+  let baseUrl = process.argv[2];
```

Comment:
Updated with 82f835c70e8378013db9998cfd7ff148e9e06881 Probably should make a higher level command that could be used for more things than just the examples in the future.